src/Bicep.Core.Samples/Files/LoadFunctions_CRLF/main.syntax.bicep

```diff
@@ -0,0 +1,78 @@
+var loadedText1 = loadTextContent('Assets/TextFile.CRLF.txt')
+var loadedText2 = sys.loadTextContent('Assets/TextFile.LF.txt')
+var loadedTextEncoding1 = loadTextContent('Assets/encoding-ascii.txt', 'us-ascii')
+var loadedTextEncoding2 = loadTextContent('Assets/encoding-utf8.txt', 'utf-8')
+var loadedTextEncoding3 = loadTextContent('Assets/encoding-utf16.txt', 'utf-16')
+var loadedTextEncoding4 = loadTextContent('Assets/encoding-utf16be.txt', 'utf-16BE')
+var loadedTextEncoding5 = loadTextContent('Assets/encoding-iso.txt', 'iso-8859-1')
+
+var loadedBinary1 = loadFileAsBase64('Assets/binary')
+var loadedBinary2 = sys.loadFileAsBase64('Assets/binary')
+
+var loadedTextInterpolation1 = 'Text: ${loadTextContent('Assets/TextFile.CRLF.txt')}'
+var loadedTextInterpolation2 = 'Text: ${loadTextContent('Assets/TextFile.LF.txt')}'
+
+var loadedTextObject1 = {
+  'text' : loadTextContent('Assets/TextFile.CRLF.txt')
+}
+var loadedTextObject2 = {
+  'text' : loadTextContent('Assets/TextFile.LF.txt')  
+}
+var loadedBinaryInObject = {
+  file: loadFileAsBase64('Assets/binary')
+}
+
+var loadedTextArray = [
+  loadTextContent('Assets/TextFile.LF.txt')
+  loadFileAsBase64('Assets/binary')
+]
+
+var loadedTextArrayInObject = {
+  'files' : [
+    loadTextContent('Assets/TextFile.CRLF.txt')
+    loadFileAsBase64('Assets/binary')
+  ]
+}
+
+var loadedTextArrayInObjectFunctions = {
+  'files' : [
+    length(loadTextContent('Assets/TextFile.CRLF.txt'))
+    sys.length(loadTextContent('Assets/TextFile.LF.txt'))
+    length(loadFileAsBase64('Assets/binary'))
+    sys.length(loadFileAsBase64('Assets/binary'))
+  ]
+}
+
+
+module module1 'modulea.bicep' = {
+  name: 'module1'
+  params: {
+    text: loadTextContent('Assets/TextFile.LF.txt')
+  }
+}
+
+module module2 'modulea.bicep' = {
+  name: 'module2'
+  params: {
+    text: loadFileAsBase64('Assets/binary')
+  }
+}
+
+var textFileInSubdirectories = loadTextContent('Assets/../Assets/path/../path/../../Assets/path/to/deep/file/../../../to/deep/file/TextFile.txt')
+var binaryFileInSubdirectories = loadFileAsBase64('Assets/../Assets/path/../path/../../Assets/path/to/deep/file/../../../to/deep/file/binary')
+
+
+//@[0:61) VariableDeclarationSyntax

```

Comment:
Something strange is up with the offset calculation in `DataSet.AddDiagsToSourceTest()` across multiple files. Do you know what might  have changed?

These annotations should be placed in-line, immediately after the symbol the refer to.

Reply:
Not intentionally, although I have some issues with tests so perhaps baseline didn't run as I'd expect it to run.

Reply:
I think the problem was that I checked out LF bicep file while the test case was specifying it's a CRLF. It's fine now.